model/workflow.go

```diff
@@ -108,7 +108,7 @@ type BaseWorkflow struct {
 	// Auth definitions can be used to define authentication information that should be applied to resources defined in the operation
 	// property of function definitions. It is not used as authentication information for the function invocation,
 	// but just to access the resource containing the function invocation information.
-	Auth AuthDefinitions `json:"auth,omitempty"`
+	Auth interface{} `json:"auth,omitempty"`
```

Comment:
interface is confusing to the user, i have give an implement example in https://github.com/spolti/sdk-go/pull/1，PTAL

Reply:
Nice :)
I think it is ready to go now.